machine-learning/Dockerfile

```
@@ -15,6 +15,34 @@ RUN mkdir /opt/armnn && \
     cd /opt/ann && \
     sh build.sh
 
+# Warning: 25GiB+ disk space required to pull this image
+# TODO: find a way to reduce the image size
+FROM rocm/dev-ubuntu-22.04:6.3.4-complete AS builder-rocm
+
+WORKDIR /code
+
+RUN apt-get update && apt-get install -y --no-install-recommends wget git python3.10-venv migraphx migraphx-dev half
```

```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends wget git python3.10-venv migraphx-dev
```
Only `migraphx-dev` is needed as the other 2 are dependencies.